_documentation/video/ofVideoGrabber.markdown

```diff
@@ -67,14 +75,14 @@ Closes the sequence grabber and de-allocates any allocated resources. Call this
 
 <!----------------------------------------------------------------------------->
 
-###void draw(x, y, w, h)
+###void draw(x, y)
 
```

Comment:
@arturoc  (Not sure if I can reply using a line-note so adding your handle here)
I actually didn't change that line; I've cut the ```void draw(x, y, w, h)``` part and moved it below ```draw(x, y)```. Instead of seeing the lines being moved, Git thinks they both have been changed. The contents of those parts is still the same as before, so I don't think this should be a problem?